CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### v3.4.0 (2023-07-11)
+
+- Make bindReporter generic over metric type ([#359](https://github.com/GoogleChrome/web-vitals/pull/359))
+- Update INP status in README ([#362](https://github.com/GoogleChrome/web-vitals/pull/362))
+- Metric type fixes ([#356](https://github.com/GoogleChrome/web-vitals/pull/356))
+- Fix selector for SVGs ([#354](https://github.com/GoogleChrome/web-vitals/pull/354))
```

Comment:
I think it'd be helpful if this clarified that it applies to selector identifiers *just* in the attribution build.

Ideally someone could glance at these CHANGELOG entries and quicky see (without clicking through to the PR) whether a change applies to them.

Reply:
Done.